### PR TITLE
check if Root.Path is Abs

### PR DESCRIPTION
**Pull request by liangchenye**
Add IsAbs check due to runtime spec changes:
https://github.com/opencontainers/runtime-spec/blob/master/config.md#root-configuration

Signed-off-by: liang chenye <liangchenye@huawei.com>

### CODE REVIEW

**Comment by liangchenye** [comment]
@mrunalp PTAL

File: cmd/oci-runtime-tool/validate.go [hunk 1]
```diff
@@ -78,7 +78,7 @@ var bundleValidateCommand = cli.Command{
 			return err
 		}
 
-		configPath := path.Join(inputPath, "config.json")
+		configPath := filepath.Join(inputPath, "config.json")
```
Comment (Mashimiao) [inline cmd/oci-runtime-tool/validate.go]: should we replace all path.* to filepath.* and remove `import "path"`?

Comment (liangchenye) [inline cmd/oci-runtime-tool/validate.go]: I think so, I modified all path in 'validate.go' by updating the PR.

Comment (mrunalp) [inline cmd/oci-runtime-tool/validate.go]: :+1:

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**